reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -197,9 +197,11 @@ public static Scheduler fromExecutorService(ExecutorService executorService, Str
 	 * tasks could be delayed due to two workers sharing the same backing thread and submitting long-running tasks,
 	 * despite another backing thread becoming idle in the meantime.
 	 *
-	 * @return a new {@link Scheduler} that dynamically create workers with an upper bound to
+	 * @return default instance of a {@link Scheduler} that dynamically creates workers with an upper bound to
 	 * the number of backing threads and after that on the number of enqueued tasks,
-	 * that reuses threads and evict idle ones
+	 * that reuses threads and evict idle ones.
+	 * Only one instance of the Scheduler will be created on the first call and cached. The same instance is returned
+	 * on subsequent calls.
```

Comment:
I also think that these additional sentences are too much for a return clause. Can you rework the javadoc so that it is instead made an additional `<p>`aragraph right before the `@return` clause? It should also be noted that these instances ignore `dispose()` calls. But they can be closed by changing the `Factory`.

```
	 * One cannot directly {@link Scheduler#dispose() dispose} the common instances, as they are cached and shared between callers.
	 * They can however be {@link #shutdownNow() shut down}, or replaced by a {@link #setFactory(Factory) change in Factory}.
```